oshmem/shmem/c/shmem_test_lock.c

```diff
@@ -28,5 +29,11 @@
 
 int shmem_test_lock(volatile long *lock)
 {
-    return _shmem_test_lock((void *)lock, sizeof(long));
+	if (oshmem_shmem_mcs_locks) {
+		if (oshmem_shmem_lock_debug)
+			printf("Calling MCS implementation\n");
```

Comment:
Please use OPAL_OUTPUT_VERBOSE or other OMPI debugging support, but not printf.

Reply:
Thanks for pointing out. I have updated and pushed a new patch.